include/cpuinfo.h

```diff
@@ -11,6 +11,7 @@
 #endif
 
 #include <stdint.h>
+#include <sys/auxv.h>
```

Comment:
Why this is needed? And as this header is included from every platform cpuinfo is supported, can you please explain part of what standard it adheres to?

Reply:
The header file sys/auxv.h was not needed in include/cpuinfo.h as I have already included in files inside src/powerpc, so I have removed it from include/cpuinfo.h
This header file is needed to use getauxval() to get cache information